library/std/src/sys/pal/windows/c/bindings.txt

```diff
@@ -1401,7 +1401,7 @@ ERROR_MARSHALL_OVERFLOW
 ERROR_MAX_SESSIONS_REACHED
 ERROR_MAX_THRDS_REACHED
 ERROR_MCA_EXCEPTION
-ERROR_MCA_OCCURED
```

Comment:
This is actually wrong, see <https://learn.microsoft.com/en-us/windows/win32/debug/system-error-codes--500-999->.

Reply:
Sorry about this. I kept only the changes in  `compiler/* `and `src/doc/rustc/*`